docs/myriad-dev.md

```diff
@@ -72,6 +90,17 @@ export MESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos.so
     <value>com.ebay.myriad.scheduler.yarn.MyriadFairScheduler</value>
     <description>One can configure other scehdulers as well from following list: com.ebay.myriad.scheduler.yarn.MyriadCapacityScheduler, com.ebay.myriad.scheduler.yarn.MyriadFifoScheduler</description>
 </property>
+
+```
+
+* Add following to ```$YARN_HOME/etc/hadoop/mapred-site.xml```:
+
+```xml
+<!--This option enable dynamic port assignment by mesos -->
+<property>
+    <name>mapred.shuffle.port</name>
+    <value>${myiad.shuffle.port}</value>
```

Comment:
+1 for that. It really helps to distinctly tell which YARN properties are myriad setting the values for.